cypress/e2e/listingDetails/completedLotteryListing.e2e.ts

```diff
@@ -51,7 +37,18 @@ describe("Listing Details for Completed Lottery Listing", () => {
   //   // there is a rogue loading issue beyond the scope of this story
   //   cy.wait(6000)
   // })
-
+  beforeEach(() => {
+    if (Cypress.env("salesforceInstanceUrl") === "https://sfhousing.my.salesforce.com") {
```

Comment:
question: Is this conditional set up as intended? Equality means that we are running with prod env vars, so it means that the fixtures will be used when running the production tests.

Reply:
yes! Because we don't want to be hitting prod salesforce anymore, we want to use fixtures for prod